fuckit.js

```diff
@@ -52,6 +52,20 @@
       window[prop] = FuckIt;
     }
   }
+  
+  FuckIt.randomConflict = function(){
+    for (var prop in window){
+        if (prop === "location")
+            continue;
+        
+        var isItFucked = Math.random();
+        var fuckedThreshold = Math.random();
+        if(isItFucked < fuckedThreshold)
```

Comment:
The odds of this being true are exactly 50%, so you might as well do isItFucked < 0.5. If you want a random threshold it should be set outside the loop. FFS.